src/ontology/components/mondo-tags.owl

```diff
@@ -13988,9 +13988,7 @@
 
     <!-- http://purl.obolibrary.org/obo/MONDO_0005489 -->
 
-    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0005489">
-        <mondo:excluded_from_qc_check rdf:resource="http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql"/>
-    </owl:Class>
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0005489"/>
```

Comment:
@sabrinatoro did you intend to make these changes on this file?

Reply:
I wanted to remove the "exclude from QC" subset from a term... I don't know why it changes this file... Thank you for catching that!

Reply:
Oooh, I think Nico added those through the mondo-tags.owl file, so if you remove them in the mondo-edit.obo file, it will remove them in the mondo-tags.owl file. Does that make sense? I usually just leave them alone. We should check with @matentzn on how to best handle that.

Reply:
@matentzn : it looks like when I removed the "subset" annotation for the term, the mondo-tags.owl file was changed (which, when thinking about it, makes sense to me). My questions is: 
1) Is this change actually ok ?
2) Should I remove this whole term from that file (since that term has no subset/tag anymore), or will the file get automatically updated ?
Thank you!

Reply:
You can updated the file yourself by running 

```
sh run.sh make components/mondo-tags.owl -B
```

Let me know if you have problems.

Reply:
done! No problem. Thank you @matentzn